src/Mod/Draft/DraftEdit.py

```diff
@@ -517,38 +554,48 @@ def endEditing(self, obj, nodeIndex, v = None):
 
     def getObjsFromSelection(self):
         "evaluate selection and returns a valid object to edit"
-        selection = FreeCADGui.Selection.getSelection()
-        self.objs = []
+        selection = Gui.Selection.getSelection()
+        self.edited_objects = []
         if len(selection) > self.maxObjects:
-            FreeCAD.Console.PrintMessage(translate("draft", 
+            App.Console.PrintMessage(translate("draft", 
                                                    "Too many objects selected, max number set to: "
                                                    + str(self.maxObjects) + "\n"))
```

Comment:
Here the `str(self.maxObjects)` should not be inside the `translate` function. The translate function should have static text only.
```py
translate("draft", 
          "Too many objects selected, max number set to: ")
          + str(self.maxObjects) + "\n"
```